sql/core/src/test/resources/sample.json

```diff
@@ -0,0 +1,2 @@
+{"a" : "2" ,"b" : "blah", "c_!@(3)":1}
```

Comment:
why add this file?

Reply:
It is needed to run a test "data source table with user specified schema". Actually we just needed to move the file. I'll remove the extra one.